config/settings/base.py

```diff
@@ -731,3 +731,17 @@
 
 # Maximum file upload size in MB
 MAX_FILE_UPLOAD_SIZE = env.int("MAX_FILE_UPLOAD_SIZE", default=5)
+
+QUESTIONNAIRE_TEXT_AREA_LIMIT_SIZE = env.int(
```

Comment:
Can we create these settings separate, having mission critical stuff and this together does not make sense.